src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -11105,121 +11105,85 @@ def test_get_trustedaccess_roles(self):
         assert len(role["rules"]) > 0
 
     @AllowLargeResponse()
-    @AKSCustomResourceGroupPreparer(
-        random_name_length=17, name_prefix="clitest", location="westus2"
-    )
-    def test_aks_trustedaccess_rolebinding(
```

Comment:
Is this test deleted in this PR? If so, please delete the corresponding recording file as well.

Reply:
deleted